tests/macros/thasargofname.nim

```diff
@@ -0,0 +1,20 @@
+discard """
+  nimout: '''
+  [true, true, true, true, true, true, true]
+  '''
+"""
+
```

Comment:
move this to `tests/stdlib/tmacros.nim`, which is the convention followed by almost all stdlib modules

Reply:
@deech  please click "resolve conversation" once you've resolved it :)